modular_skyrat/modules/novaya_ert/code/outfit.dm

```diff
@@ -1,19 +1,21 @@
 /datum/outfit/centcom/ert/nri
 	name = "Novaya Rossiyskaya Imperiya Soldier"
-	head = /obj/item/clothing/head/helmet/space/hev_suit/nri
+	head = null
 	glasses = /obj/item/clothing/glasses/night
 	ears = /obj/item/radio/headset/headset_cent/alt/with_key
 	mask = /obj/item/clothing/mask/gas/hecu2
 	uniform = /obj/item/clothing/under/costume/nri
-	suit = /obj/item/clothing/suit/space/hev_suit/nri
-	suit_store = /obj/item/gun/ballistic/automatic/akm
+	suit = null
+	suit_store = null
 	gloves = /obj/item/clothing/gloves/combat
 	belt = /obj/item/storage/belt/military/nri/soldier
-	back = /obj/item/storage/backpack/duffelbag/syndie/nri
-	backpack_contents = list(/obj/item/storage/box/nri_survival_pack, /obj/item/storage/medkit/emergency, /obj/item/ammo_box/magazine/m9mm_aps, /obj/item/advanced_choice_beacon/nri/heavy, /obj/item/beamout_tool, /obj/item/crucifix)
+	back = /obj/item/mod/control/pre_equipped/frontline/ert
+	backpack_contents = list(/obj/item/storage/box/nri_survival_pack, /obj/item/storage/medkit/emergency, /obj/item/ammo_box/magazine/m9mm_aps, /obj/item/advanced_choice_beacon/nri/heavy, /obj/item/beamout_tool, /obj/item/crucifix, /obj/item/reagent_containers/cup/glass/waterbottle/large/cryptobiolin)
```

Comment:
Make these multi-line lists please, and end the last element with a trailing comma, to lessen the impact on the git blame of any additions made to those lists in the future.